ko/admin/control.rst

```diff
@@ -937,10 +942,15 @@ CCI 드라이버를 사용하여 C로 프로그램을 작성할 때는 에러 
     @ cubrid broker start
     ++ cubrid broker is running.
 
+.. warning::
+    On Linux systems, the number of file descriptors (fd) required to run shards is moderately higher than SHARD_MAX_CLIENTS set in subrid_broker.conf, so when limiting the number of fds with "ulimit -n", it should be set appropriately larger than SHARD_MAX_CLIENTS.
```

Comment:
It seems like an unnecessary sentence.

Reply:
I made a mistake while adding content. I'll fix it right away.